Gruntfile.js

```diff
@@ -143,31 +141,54 @@ module.exports = function (grunt) {
 
   grunt.loadNpmTasks('grunt-eslint');
   grunt.loadNpmTasks('grunt-karma');
-  grunt.loadNpmTasks('grunt-browserify');
   grunt.loadNpmTasks('grunt-contrib-uglify');
 
   grunt.registerTask('test', [
     'eslint',
-    'browserify',
+    'compatibilityWithWebpackProd',
+    'buildWebpack',
     'uglify',
     'karma:unit',
     'karma:fuzz',
     'karma:functional',
     'karma:compatibilityWithVanillaScript',
     'karma:compatibilityWithVanillaWorker',
-    'karma:compatibilityWithBrowserify',
-    'karma:compatibilityWithWebpack'
+    'karma:compatibilityWithBrowserify'
   ]);
 
   grunt.registerTask('test:unit', [
     'eslint',
-    'browserify',
+    'buildWebpack',
     'uglify',
     'karma:unit'
   ]);
 
-  grunt.registerTask('benchmark', ['browserify', 'uglify', 'karma:benchmark']);
+  grunt.registerTask('compatibilityWithWebpackProd', ['buildWebpackProd', 'uglify', 'karma:compatibilityWithWebpackProd'])
 
-  grunt.registerTask('build', ['eslint', 'browserify', 'uglify']);
+  grunt.registerTask('benchmark', ['buildWebpack', 'uglify', 'karma:benchmark']);
 
+  grunt.registerTask('build', ['eslint', 'buildWebpack', 'uglify']);
+
+  grunt.registerTask('buildWebpack', 'create dist using Webpack', function() {
```

Comment:
I didn't want to add another grunt dependency, instead Im just using the webpack bin. Hope thats fine 🤞